.env

```diff
@@ -1,10 +1,11 @@
 COMPOSE_PROJECT_NAME=xpub
 XPUB_VERSION=8526f064363e107edefb5c118bbd51dd1b2e72cd
+REPORTER_VERSION=47808c55b705d40e4d6c0356a4614027e630def8
```

Comment:
assume we will need a Jenkins job to update this?

Reply:
Or can we keep it manual?

Reply:
I don't think the overhead is worth it for xpub, hopefully not long now :crossed_fingers: